website/content/docs/api-gateway.mdx

```diff
@@ -38,7 +38,7 @@ Your datacenter must meet the following requirements prior to configuring the Co
 
    </CodeBlockConfig>
 
-1. Create a values file for your Consul server agents that contains the following parameters:
+1. Create a `values.yaml` file for your Consul API Gateway deployment that contains the following configuration through the Consul Helm chart. See [Helm Chart Configuration - apigateway](https://www.consul.io/docs/k8s/helm#apigateway) for more available options on how to configure your Consul API Gateway deployment via the Helm chart. 
```

Comment:
```suggestion
1. Create a `values.yaml` file for your Consul API Gateway deployment. Copy the content below into the `values.yaml` file. The `values.yaml` will be used by the Consul Helm chart. See [Helm Chart Configuration - apigateway](https://www.consul.io/docs/k8s/helm#apigateway) for more available options on how to configure your Consul API Gateway deployment via the Helm chart. 
```
The first sentence is a run-on sentence, so I broke it up into three sentences that makes it easier for the reader to follow along.